### PR TITLE
rtt updates for Srautas Gorge

### PR DESCRIPTION
**Pull request by terrapin47**

### CODE REVIEW

File: QUEST_LV_0100.tsv [hunk 1]
```diff
@@ -5636,36 +5636,36 @@ QUEST_LV_0100_20150317_005635	$Destroy all the magic suppressors that Helgasercl
 QUEST_LV_0100_20150317_005636	$Find and defeat Helgasercle who invaded the Mage Tower.
 QUEST_LV_0100_20150317_005637	Talk to Guard Gilbert
 QUEST_LV_0100_20150317_005638	{memo X}
-QUEST_LV_0100_20150317_005639	Defeat Grummer and Seedmia
-QUEST_LV_0100_20150317_005640	Gilbert wants you to defeat the monsters that destroyed the cable car. Defeat them in Cholas Crooked Road and tell Gilbert about it. 
+QUEST_LV_0100_20150317_005639	Defeat Grummers and Zignuts
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:5643]: Why change it from Seedmia to Zignut? It says Seedmia in the original. 
Error on IMC's side?

File: QUEST_LV_0100.tsv [hunk 2]
```diff
@@ -6600,7 +6600,7 @@ QUEST_LV_0100_20150323_006599	Morkus Jonas says you can listen to the next story
 QUEST_LV_0100_20150323_006600	Go and meet Rolandas Jonas in Akmens Ridge.
 QUEST_LV_0100_20150323_006601	{memo X}Goddess Saule says Bramble of Kvailas Forest is in Thorn Forest. Pass through Gate Route and Spine Heart Dale to Kvailas Forest to find the revelation. 
 QUEST_LV_0100_20150323_006602	Kvailas Forest
-QUEST_LV_0100_20150323_006603	Move to Gele Plateau
+QUEST_LV_0100_20150323_006603	Travel to Gele Plateau
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:6607]: Can you give me the context of this piece of text?

File: QUEST_LV_0100.tsv [hunk 3]
```diff
@@ -7554,7 +7554,7 @@ QUEST_LV_0100_20150714_007553	$Hand them over to Guard Molly
 QUEST_LV_0100_20150714_007554	$You've obtained all the materials that are needed to make Stringers. Hand them over to Guard Molly.
 QUEST_LV_0100_20150714_007555	$You've obtained all the materials that are needed to make Stringers. Ask Guard Molly about what to do next.
 QUEST_LV_0100_20150714_007556	$Set the Stringers
-QUEST_LV_0100_20150714_007557	$To prevent Pantos from straying to other regions, please set the Stringers in Srautas Canyon and Nefritas Cliffs.
+QUEST_LV_0100_20150714_007557	$To prevent Pantos from straying to other regions, please set the Stringers in Srautas Gorge and Nefritas Cliffs.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Nefritas Cliffs -> Nefritas Cliff

File: QUEST_LV_0100.tsv (hunk 1 above)
Comment (terrapin47) [inline QUEST_LV_0100.tsv:5643]: Yup.  Seedmia's don't even spawn on this map :P

File: QUEST_LV_0100.tsv (hunk 2 above)
Comment (terrapin47) [inline QUEST_LV_0100.tsv:6607]: The Paladin Master is waiting for you in Gele Plateau, after you've received the revelation in the mines.  You need to travel across all the maps from the Miners' Village to Gele Plateau and meet him.

File: QUEST_LV_0100.tsv (hunk 3 above)
Comment (terrapin47) [inline QUEST_LV_0100.tsv]: I came across this line while changing instances of 'Srautas Canyon', any clue where this quest happens?  I dunno what 'Stringers' are, and would like to do the quest to get some context.

Comment (Sunaries) [inline QUEST_LV_0100.tsv]: It's probably an item.

**Comment by terrapin47** [comment]
:+1:

**Comment by Sunaries** [comment]
Good to merge.

**Comment by imcgames** [comment]
:+1:

**Pull request merged**